pom.xml

```diff
@@ -343,6 +343,17 @@
             <version>2.1</version>
             <scope>runtime</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-artifact</artifactId>
+            <version>3.0.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.powermock</groupId>
```

Comment:
@gumbelmj Ok. So let's get rid of it as I said, and modify the tests as I suggested.